### PR TITLE
Add foreign architectures for package manager

### PR DESCRIPTION
**Pull request by ehargitt**
This is my first PR ever, so forgive me if I screw it up 😬. I added the foreign architecture just for the `gcc49-armv7hf` image for now so you could see what that would look like. I also added a statement to the `test.sh` script, but it looks like that doesn't get run for now anyway. The global variables in the `.travis.yaml` file were overriding the variables for the build plan in my Travis setup and causing the builds to fail. Consider removing them so others can fork and test in their own Travis builds. Anyway, let me know what you think and I could help do this for the other images.

### CODE REVIEW

**Comment by ehargitt** [comment]
Also, all the Travis build pass except for `clang4-x86`.

File: .travis.yml [hunk 1]
```diff
@@ -6,11 +6,6 @@ services:
   - docker
 
 env:
-  global:
```
Comment (uilianries) [inline .travis.yml]: why did you remove the env vars?

Comment (ehargitt) [reply .travis.yml]: Because the variables in `.travis.yaml` override the variables set in my Travis build. The commit message has more info. Idk how to link a commit, but its commit `a3e55cb`.

**Comment by lasote** [comment]
LGTM. About the env vars, only the DOCKER_UPLOAD is not defaulted in the build.py but I have just set it in the travis build conf. 
Thanks!

**Comment by ehargitt** [comment]
@lasote It looks like `DOCKER_UPLOAD` does have a default value of `"false"` in my branch. Do I need to put the env vars back in the `.travis.yml` file?

**Comment by lasote** [comment]
No, when it is merged it will run a new build with upload set to True.
Are you adjusting the foreign archs for the rest of cross build images or this is all?

**Comment by lasote** [comment]
Ping @uilianries

**Comment by uilianries** [comment]
@lasote You could remove, yes. Those script are good for who don't have python on machine, but I thing any machine could have python installed.

I'll remove on python3 branch.

@ehargitt I think your PR is good to go.

**Comment by uilianries** [comment]
@lasote you will need to update travis settings, including DOCKER_UPLOAD, DOCKER_USERNAME and DOCKER_LOGIN_USERNAME.

**Comment by ehargitt** [comment]
Well, I still have to add the foreign architectures to the rest of the Docker images. I'll let you guys know when it's finished.

**Comment by ehargitt** [comment]
@uilianries My apologies. I will fix these issues and start using the Docker linter.

**Comment by uilianries** [comment]
@ehargitt We are using hadolint, actually there is no official Docker linter yet. Hadolint is not very updated, but it's the less worst that I found. We are running it for each image, but is not breaking our builds, only showing messages on log.

**Comment by ehargitt** [comment]
I'm having an issue with gcc49-armv7hf where I cannot install `-dev` packages. For example, I can install `libxml2:armhf` ok, but not `libxml2-dev:armhf`. Here is the message I get:
```bash
conan@8ccf1b77bcd6:~$ sudo apt-get install libxml2-dev:armhf
Reading package lists... Done
Building dependency tree
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 libxml2-dev:armhf : Depends: libicu-dev:armhf but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
```
As I drill down through the unmet dependencies, I come across `binutils:armhf`. This is what I get when I try to install it:
```bash
conan@8ccf1b77bcd6:~$ sudo apt-get install binutils:armhf
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following packages were automatically installed and are no longer required:
  autoconf automake autopoint autotools-dev bzip2 cpp-4.9 ecj ecj-gcj gcc-4.9-base gcc-5-cross-base gcj-5-jre-headless gcj-5-jre-lib gettext gettext-base groff-base intltool-debian java-common libasan1 libasan1-armhf-cross libasan2
  libasound2 libasound2-data libasprintf0v5 libatomic1 libatomic1-armhf-cross libc6-armel-armhf-cross libc6-armel-cross libc6-armhf-cross libc6-dev-armel-armhf-cross libc6-dev-armel-cross libc6-dev-armhf-cross libcc1-0 libcilkrts5
  libcloog-isl4 libcroco3 libdpkg-perl libecj-java libecj-java-gcj libgcc-4.9-dev libgcc-4.9-dev-armhf-cross libgcc-5-dev libgcc1-armhf-cross libgcj-bc libgcj-common libgcj16 libgfortran-4.9-dev-armhf-cross libgfortran3-armhf-cross
  libglib2.0-0 libgo5-armhf-cross libgomp1 libgomp1-armhf-cross libicu55 libisl13 libitm1 liblsan0 libmpx0 libobjc-4.9-dev-armhf-cross libobjc4-armhf-cross libphobos-4.9-dev libphobos-4.9-dev-armhf-cross libpipeline1 libquadmath0
  libsfasan1-armhf-cross libsfatomic1-armhf-cross libsfgcc-4.9-dev-armhf-cross libsfgcc1-armhf-cross libsfgfortran-4.9-dev-armhf-cross libsfgfortran3-armhf-cross libsfgomp1-armhf-cross libsfobjc-4.9-dev-armhf-cross
  libsfobjc4-armhf-cross libsfstdc++-4.9-dev-armhf-cross libsfstdc++6-armhf-cross libsfubsan0-armhf-cross libsigsegv2 libstdc++-4.9-dev libstdc++-4.9-dev-armhf-cross libstdc++-5-dev libstdc++6-armhf-cross libtsan0 libubsan0
  libubsan0-armhf-cross libunistring0 libxml2 linux-libc-dev-armel-cross linux-libc-dev-armhf-cross m4 make man-db po-debconf xz-utils
Use 'apt-get autoremove' to remove them.
Suggested packages:
  binutils-doc:armhf
The following packages will be REMOVED:
  binutils binutils-arm-linux-gnueabihf debhelper dpkg-dev g++-4.9 g++-4.9-arm-linux-gnueabihf g++-4.9-multilib-arm-linux-gnueabihf gcc-4.9 gcc-4.9-arm-linux-gnueabihf gcc-4.9-multilib-arm-linux-gnueabihf gccgo-4.9-arm-linux-gnueabihf
  gcj-4.9 gcj-4.9-arm-linux-gnueabihf gdc-4.9 gdc-4.9-arm-linux-gnueabihf gfortran-4.9-arm-linux-gnueabihf gfortran-4.9-multilib-arm-linux-gnueabihf gobjc++-4.9-arm-linux-gnueabihf gobjc++-4.9-multilib-arm-linux-gnueabihf
  gobjc-4.9-arm-linux-gnueabihf gobjc-4.9-multilib-arm-linux-gnueabihf pkg-config
The following NEW packages will be installed:
  binutils:armhf
0 upgraded, 1 newly installed, 22 to remove and 0 not upgraded.
Need to get 2304 kB of archives.
After this operation, 252 MB disk space will be freed.
Do you want to continue? [Y/n]
```
You can see that it wants to remove gcc 4.9 and its kin, which obviously would defeat the purpose of this Docker image. Do you guys have any ideas of how to work around this? Maybe install gcc 4.9 without the package manager?

**Comment by uilianries** [comment]
no idea, but we have libxml2 packaged on Bintray.

Maybe you could avoid system packages and install by Conan. both gcc-48 and gcc-49 are using old distros flavors and it's a pain to maintain because there is no support from ubuntu because are EOL.

In my experience you will waste more time trying to do black magic just to solve a distro limitation than investing time to create your product.

**Comment by ericLemanissier** [comment]
I'd be very interested to see this PR move forward, because it is a blocking point for cross compilation of the Qt recipe. If no simple solution exist for gcc 4.9, maybe we should split the PR and do all the other before? Can I help ?

**Comment by ehargitt** [comment]
Sorry, I've had this PR on the back burner. @ericLemanissier yes, I would be interested in sharing this work. I lost momentum on this because my company is using gcc 4.9 and @uilianries is probably right about trying to solve the distro limitation. The only long term solution I see is switching to a different distro for the base image (idk which one, maybe Alpine?). All the Ubuntu versions eventually get EOLed and they change the location of the packages, which is the source of the pain. What do you think @ericLemanissier?

**Comment by ehargitt** [comment]
@uilianries I don't really see it as a waste of time to solve this. Conan itself supports the use of the package manager. Shouldn't the tools that are supposed to work with Conan also support it?

**Comment by uilianries** [comment]
the only problem switching the base distro is the libc version. we could change the base version we could break many users using our arm image for gcc 4.9

is it possible install all things using alpine? is there a better option? centos?

**Comment by ehargitt** [comment]
@uilianries you're right about libc; Alpine uses musl-libc, which is binary incompatible with glibc 2.x (libc6). The next option to consider is Debian. Ubuntu is based on Debian, and Debian uses libc6. Unlike Ubuntu, Debian's archives aren't split, meaning you don't normally have to add/qualify apt sources for your foreign architecture (`deb [arch=armel,armhf] http://ports.ubuntu.com/ubuntu-ports quantal main universe` for example). I confirmed that it works "out of the box" by pulling the `debian:stretch-slim` image, adding my arch, updating, and installing a package (output below). Note that this doesn't solve the issue of package locations changing over time; Debian is committed to supporting each stable release for at least 5 years, and after that the packages may move to the archive section. Let me know what you think of using Debian as the new base image.

```bash
PS C:\Windows\system32> docker run -it --rm debian:stretch-slim
root@69661a8dc0dd:/# dpkg --add-architecture armhf
root@69661a8dc0dd:/# apt-get update
Get:1 http://security-cdn.debian.org/debian-security stretch/updates InRelease [94.3 kB]
Ign:2 http://cdn-fastly.deb.debian.org/debian stretch InRelease
Get:5 http://security-cdn.debian.org/debian-security stretch/updates/main amd64 Packages [459 kB]
Get:3 http://cdn-fastly.deb.debian.org/debian stretch-updates InRelease [91.0 kB]
Get:4 http://cdn-fastly.deb.debian.org/debian stretch Release [118 kB]
Get:6 http://cdn-fastly.deb.debian.org/debian stretch Release.gpg [2434 B]
Get:7 http://security-cdn.debian.org/debian-security stretch/updates/main armhf Packages [444 kB]
Get:8 http://cdn-fastly.deb.debian.org/debian stretch-updates/main armhf Packages [5108 B]
Get:9 http://cdn-fastly.deb.debian.org/debian stretch-updates/main amd64 Packages [5152 B]
Get:10 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 Packages [7089 kB]
Get:11 http://cdn-fastly.deb.debian.org/debian stretch/main armhf Packages [6919 kB]
Fetched 15.2 MB in 5s (2620 kB/s)
Reading package lists... Done
root@69661a8dc0dd:/# apt-get install libxml2-dev:armhf
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following additional packages will be installed:
  gcc-6-base:armhf icu-devtools libasan3:armhf libatomic1:armhf libc-dev-bin libc6:armhf libc6-dev:armhf libgcc-6-dev:armhf libgcc1:armhf libgomp1:armhf libicu-dev:armhf libicu57 libicu57:armhf liblzma5:armhf libstdc++-6-dev:armhf
  libstdc++6:armhf libubsan0:armhf libxml2:armhf linux-libc-dev:armhf manpages manpages-dev sgml-base xml-core zlib1g:armhf
Suggested packages:
  glibc-doc:armhf libc-l10n:armhf locales:armhf icu-doc:armhf libstdc++-6-doc:armhf pkg-config:armhf man-browser sgml-base-doc debhelper
The following NEW packages will be installed:
  gcc-6-base:armhf icu-devtools libasan3:armhf libatomic1:armhf libc-dev-bin libc6:armhf libc6-dev:armhf libgcc-6-dev:armhf libgcc1:armhf libgomp1:armhf libicu-dev:armhf libicu57 libicu57:armhf liblzma5:armhf libstdc++-6-dev:armhf
  libstdc++6:armhf libubsan0:armhf libxml2:armhf libxml2-dev:armhf linux-libc-dev:armhf manpages manpages-dev sgml-base xml-core zlib1g:armhf
0 upgraded, 25 newly installed, 0 to remove and 0 not upgraded.
Need to get 45.5 MB of archives.
After this operation, 187 MB of additional disk space will be used.
Do you want to continue? [Y/n] y
Get:1 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 sgml-base all 1.29 [14.8 kB]
Get:2 http://cdn-fastly.deb.debian.org/debian stretch/main armhf gcc-6-base armhf 6.3.0-18+deb9u1 [180 kB]
Get:3 http://cdn-fastly.deb.debian.org/debian stretch/main armhf libgcc1 armhf 1:6.3.0-18+deb9u1 [38.5 kB]
Get:4 http://cdn-fastly.deb.debian.org/debian stretch/main armhf libc6 armhf 2.24-11+deb9u3 [2340 kB]
Get:5 http://cdn-fastly.deb.debian.org/debian stretch/main armhf liblzma5 armhf 5.2.2-1.2+b1 [226 kB]
Get:6 http://cdn-fastly.deb.debian.org/debian stretch/main armhf zlib1g armhf 1:1.2.8.dfsg-5 [82.4 kB]
Get:7 http://cdn-fastly.deb.debian.org/debian stretch/main armhf libstdc++6 armhf 6.3.0-18+deb9u1 [346 kB]
Get:8 http://cdn-fastly.deb.debian.org/debian stretch/main armhf libicu57 armhf 57.1-6+deb9u2 [7449 kB]
Get:9 http://cdn-fastly.deb.debian.org/debian stretch/main armhf libxml2 armhf 2.9.4+dfsg1-2.2+deb9u2 [825 kB]
Get:10 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 manpages all 4.10-2 [1222 kB]
Get:11 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 libicu57 amd64 57.1-6+deb9u2 [7700 kB]
Get:12 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 icu-devtools amd64 57.1-6+deb9u2 [178 kB]
Get:13 http://cdn-fastly.deb.debian.org/debian stretch/main armhf libasan3 armhf 6.3.0-18+deb9u1 [284 kB]
Get:14 http://cdn-fastly.deb.debian.org/debian stretch/main armhf libatomic1 armhf 6.3.0-18+deb9u1 [6010 B]
Get:15 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 libc-dev-bin amd64 2.24-11+deb9u3 [258 kB]
Get:16 http://cdn-fastly.deb.debian.org/debian stretch/main armhf linux-libc-dev armhf 4.9.130-2 [1350 kB]
Get:17 http://cdn-fastly.deb.debian.org/debian stretch/main armhf libc6-dev armhf 2.24-11+deb9u3 [1995 kB]
Get:18 http://cdn-fastly.deb.debian.org/debian stretch/main armhf libgomp1 armhf 6.3.0-18+deb9u1 [64.6 kB]
Get:19 http://cdn-fastly.deb.debian.org/debian stretch/main armhf libubsan0 armhf 6.3.0-18+deb9u1 [92.9 kB]
Get:20 http://cdn-fastly.deb.debian.org/debian stretch/main armhf libgcc-6-dev armhf 6.3.0-18+deb9u1 [531 kB]
Get:21 http://cdn-fastly.deb.debian.org/debian stretch/main armhf libstdc++-6-dev armhf 6.3.0-18+deb9u1 [1487 kB]
Get:22 http://cdn-fastly.deb.debian.org/debian stretch/main armhf libicu-dev armhf 57.1-6+deb9u2 [15.9 MB]
Get:23 http://cdn-fastly.deb.debian.org/debian stretch/main armhf libxml2-dev armhf 2.9.4+dfsg1-2.2+deb9u2 [747 kB]
Get:24 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 manpages-dev all 4.10-2 [2145 kB]
Get:25 http://cdn-fastly.deb.debian.org/debian stretch/main amd64 xml-core all 0.17 [23.2 kB]
Fetched 45.5 MB in 13s (3417 kB/s)
debconf: delaying package configuration, since apt-utils is not installed
Selecting previously unselected package sgml-base.
(Reading database ... 6315 files and directories currently installed.)
Preparing to unpack .../00-sgml-base_1.29_all.deb ...
Unpacking sgml-base (1.29) ...
Selecting previously unselected package gcc-6-base:armhf.
Preparing to unpack .../01-gcc-6-base_6.3.0-18+deb9u1_armhf.deb ...
Unpacking gcc-6-base:armhf (6.3.0-18+deb9u1) ...
Selecting previously unselected package libgcc1:armhf.
Preparing to unpack .../02-libgcc1_1%3a6.3.0-18+deb9u1_armhf.deb ...
Unpacking libgcc1:armhf (1:6.3.0-18+deb9u1) ...
Selecting previously unselected package libc6:armhf.
Preparing to unpack .../03-libc6_2.24-11+deb9u3_armhf.deb ...
debconf: unable to initialize frontend: Dialog
debconf: (No usable dialog-like program is installed, so the dialog based frontend cannot be used. at /usr/share/perl5/Debconf/FrontEnd/Dialog.pm line 76.)
debconf: falling back to frontend: Readline
debconf: unable to initialize frontend: Readline
debconf: (Can't locate Term/ReadLine.pm in @INC (you may need to install the Term::ReadLine module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.24.1 /usr/local/share/perl/5.24.1 /usr/lib/x86_64-linux-gnu/perl5/5.24 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.24 /usr/share/perl/5.24 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base .) at /usr/share/perl5/Debconf/FrontEnd/Readline.pm line 7.)
debconf: falling back to frontend: Teletype
Unpacking libc6:armhf (2.24-11+deb9u3) ...
Selecting previously unselected package liblzma5:armhf.
Preparing to unpack .../04-liblzma5_5.2.2-1.2+b1_armhf.deb ...
Unpacking liblzma5:armhf (5.2.2-1.2+b1) ...
Selecting previously unselected package zlib1g:armhf.
Preparing to unpack .../05-zlib1g_1%3a1.2.8.dfsg-5_armhf.deb ...
Unpacking zlib1g:armhf (1:1.2.8.dfsg-5) ...
Selecting previously unselected package libstdc++6:armhf.
Preparing to unpack .../06-libstdc++6_6.3.0-18+deb9u1_armhf.deb ...
Unpacking libstdc++6:armhf (6.3.0-18+deb9u1) ...
Selecting previously unselected package libicu57:armhf.
Preparing to unpack .../07-libicu57_57.1-6+deb9u2_armhf.deb ...
Unpacking libicu57:armhf (57.1-6+deb9u2) ...
Selecting previously unselected package libxml2:armhf.
Preparing to unpack .../08-libxml2_2.9.4+dfsg1-2.2+deb9u2_armhf.deb ...
Unpacking libxml2:armhf (2.9.4+dfsg1-2.2+deb9u2) ...
Selecting previously unselected package manpages.
Preparing to unpack .../09-manpages_4.10-2_all.deb ...
Unpacking manpages (4.10-2) ...
Selecting previously unselected package libicu57:amd64.
Preparing to unpack .../10-libicu57_57.1-6+deb9u2_amd64.deb ...
Unpacking libicu57:amd64 (57.1-6+deb9u2) ...
Selecting previously unselected package icu-devtools.
Preparing to unpack .../11-icu-devtools_57.1-6+deb9u2_amd64.deb ...
Unpacking icu-devtools (57.1-6+deb9u2) ...
Selecting previously unselected package libasan3:armhf.
Preparing to unpack .../12-libasan3_6.3.0-18+deb9u1_armhf.deb ...
Unpacking libasan3:armhf (6.3.0-18+deb9u1) ...
Selecting previously unselected package libatomic1:armhf.
Preparing to unpack .../13-libatomic1_6.3.0-18+deb9u1_armhf.deb ...
Unpacking libatomic1:armhf (6.3.0-18+deb9u1) ...
Selecting previously unselected package libc-dev-bin.
Preparing to unpack .../14-libc-dev-bin_2.24-11+deb9u3_amd64.deb ...
Unpacking libc-dev-bin (2.24-11+deb9u3) ...
Selecting previously unselected package linux-libc-dev:armhf.
Preparing to unpack .../15-linux-libc-dev_4.9.130-2_armhf.deb ...
Unpacking linux-libc-dev:armhf (4.9.130-2) ...
Selecting previously unselected package libc6-dev:armhf.
Preparing to unpack .../16-libc6-dev_2.24-11+deb9u3_armhf.deb ...
Unpacking libc6-dev:armhf (2.24-11+deb9u3) ...
Selecting previously unselected package libgomp1:armhf.
Preparing to unpack .../17-libgomp1_6.3.0-18+deb9u1_armhf.deb ...
Unpacking libgomp1:armhf (6.3.0-18+deb9u1) ...
Selecting previously unselected package libubsan0:armhf.
Preparing to unpack .../18-libubsan0_6.3.0-18+deb9u1_armhf.deb ...
Unpacking libubsan0:armhf (6.3.0-18+deb9u1) ...
Selecting previously unselected package libgcc-6-dev:armhf.
Preparing to unpack .../19-libgcc-6-dev_6.3.0-18+deb9u1_armhf.deb ...
Unpacking libgcc-6-dev:armhf (6.3.0-18+deb9u1) ...
Selecting previously unselected package libstdc++-6-dev:armhf.
Preparing to unpack .../20-libstdc++-6-dev_6.3.0-18+deb9u1_armhf.deb ...
Unpacking libstdc++-6-dev:armhf (6.3.0-18+deb9u1) ...
Selecting previously unselected package libicu-dev:armhf.
Preparing to unpack .../21-libicu-dev_57.1-6+deb9u2_armhf.deb ...
Unpacking libicu-dev:armhf (57.1-6+deb9u2) ...
Selecting previously unselected package libxml2-dev:armhf.
Preparing to unpack .../22-libxml2-dev_2.9.4+dfsg1-2.2+deb9u2_armhf.deb ...
Unpacking libxml2-dev:armhf (2.9.4+dfsg1-2.2+deb9u2) ...
Selecting previously unselected package manpages-dev.
Preparing to unpack .../23-manpages-dev_4.10-2_all.deb ...
Unpacking manpages-dev (4.10-2) ...
Selecting previously unselected package xml-core.
Preparing to unpack .../24-xml-core_0.17_all.deb ...
Unpacking xml-core (0.17) ...
Setting up manpages (4.10-2) ...
Setting up gcc-6-base:armhf (6.3.0-18+deb9u1) ...
Setting up linux-libc-dev:armhf (4.9.130-2) ...
Setting up sgml-base (1.29) ...
Setting up libicu57:amd64 (57.1-6+deb9u2) ...
Processing triggers for libc-bin (2.24-11+deb9u3) ...
Setting up icu-devtools (57.1-6+deb9u2) ...
Setting up libc-dev-bin (2.24-11+deb9u3) ...
Setting up xml-core (0.17) ...
Setting up manpages-dev (4.10-2) ...
Setting up libc6:armhf (2.24-11+deb9u3) ...
debconf: unable to initialize frontend: Dialog
debconf: (No usable dialog-like program is installed, so the dialog based frontend cannot be used. at /usr/share/perl5/Debconf/FrontEnd/Dialog.pm line 76.)
debconf: falling back to frontend: Readline
debconf: unable to initialize frontend: Readline
debconf: (Can't locate Term/ReadLine.pm in @INC (you may need to install the Term::ReadLine module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.24.1 /usr/local/share/perl/5.24.1 /usr/lib/x86_64-linux-gnu/perl5/5.24 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.24 /usr/share/perl/5.24 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base .) at /usr/share/perl5/Debconf/FrontEnd/Readline.pm line 7.)
debconf: falling back to frontend: Teletype
Setting up libc6-dev:armhf (2.24-11+deb9u3) ...
Setting up libgcc1:armhf (1:6.3.0-18+deb9u1) ...
Setting up libgomp1:armhf (6.3.0-18+deb9u1) ...
Setting up libatomic1:armhf (6.3.0-18+deb9u1) ...
Setting up liblzma5:armhf (5.2.2-1.2+b1) ...
Setting up zlib1g:armhf (1:1.2.8.dfsg-5) ...
Setting up libstdc++6:armhf (6.3.0-18+deb9u1) ...
Setting up libasan3:armhf (6.3.0-18+deb9u1) ...
Setting up libubsan0:armhf (6.3.0-18+deb9u1) ...
Setting up libicu57:armhf (57.1-6+deb9u2) ...
Setting up libxml2:armhf (2.9.4+dfsg1-2.2+deb9u2) ...
Setting up libgcc-6-dev:armhf (6.3.0-18+deb9u1) ...
Setting up libstdc++-6-dev:armhf (6.3.0-18+deb9u1) ...
Setting up libicu-dev:armhf (57.1-6+deb9u2) ...
Setting up libxml2-dev:armhf (2.9.4+dfsg1-2.2+deb9u2) ...
Processing triggers for sgml-base (1.29) ...
Processing triggers for libc-bin (2.24-11+deb9u3) ...
root@69661a8dc0dd:/#
```

**Comment by ericLemanissier** [comment]
I guess this can be closes too, now that #101 is merged

**Comment by uilianries** [comment]
Closing it now
